integration/grpc_test.go

```diff
@@ -0,0 +1,134 @@
+package integration
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"net"
+	"os"
+	"time"
+
+	"github.com/containous/traefik/integration/helloworld"
+	"github.com/containous/traefik/integration/try"
+	"github.com/go-check/check"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+var LocalhostCert = []byte(`-----BEGIN CERTIFICATE-----
```

Comment:
I didn't find any cert on 127.0.0.1. And I really need a local cert for contacting backend on 127.0.0.1 in a secure way.

May be I can put all the certs in a shared directory for futur reuse ? @dtomcej WDYT ?